### PR TITLE
poco: fix clang build

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-poco/PKGBUILD [hunk 1]
```diff
@@ -67,9 +67,15 @@ build() {
     ;;
   esac
 
+  case {$MSYSTEM} in
+    CLANG*)
+      makedepends+=("binutils")
```
Comment (mati865) [inline mingw-w64-poco/PKGBUILD:72]: Why?

Comment (raedrizqie) [reply mingw-w64-poco/PKGBUILD:72]: for windmc.exe

Comment (jeremyd2019) [reply mingw-w64-poco/PKGBUILD:72]: > for windmc.exe

https://github.com/mstorsjo/llvm-mingw/issues/134

Comment (mati865) [reply mingw-w64-poco/PKGBUILD:72]: Thanks, GitHub for some reason doesn't let me resolve this conversation.

File: mingw-w64-poco/012-fix-mingw-redefinition.patch [hunk 2]
```diff
@@ -4,7 +4,7 @@
  #include "Poco/Net/ICMPPacketImpl.h"
  #include <cstddef>
  
-+#ifdef __MINGW32__
++#if defined(__MINGW32__) || defined(__clang__)
```
Comment (lazka) [inline mingw-w64-poco/012-fix-mingw-redefinition.patch:7]: why is this needed?

Comment (raedrizqie) [reply mingw-w64-poco/012-fix-mingw-redefinition.patch:7]: i think clang doesn't defines __MINGW32__ , right?

Comment (lazka) [reply mingw-w64-poco/012-fix-mingw-redefinition.patch:7]:
```
user@desktop CLANG64 ~
$ cat a.cpp
__MINGW32__
user@desktop CLANG64 ~
$ clang -E a.cpp
# 1 "a.cpp"
# 1 "<built-in>" 1
# 1 "<built-in>" 3
# 403 "<built-in>" 3
# 1 "<command line>" 1
# 1 "<built-in>" 2
# 1 "a.cpp" 2
1
```
